.gitignore

```diff
@@ -1,7 +1,8 @@
 .dart_tool/
 .DS_Store
 .gradle/
-.idea/
+.idea/workspace.xml
+.idea/libraries/Dart_Packages.xml
```

Comment:
How about Dart_*.xml instead? (Dart_SDK.xml)

Reply:
I don't see that file for me locally, but am happy to add it to the ignore file.

Although, is yours older? It could be that if you delete your file, it won't be re-created.

Reply:
Mine is from Jan 21. Probably the last time I ran the plugin tool tests from IntelliJ (just guessing).